docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -63,11 +67,14 @@ You can also specify:
   language to the GitHub code scanning API. For more information about this use case,
   see `Configuring CodeQL CLI in your CI system <https://docs.github.com/en/code-security/secure-coding/using-codeql-code-scanning-with-your-existing-ci-system/configuring-codeql-cli-in-your-ci-system>`__ in the GitHub documentation.
 
-- ``--sarif-add-query-help``: (supported in version 2.7.1 onwards) adds any custom query help written 
-   in markdown to SARIF files (v2.1.0 or later) generated by the analysis. Query help stored in ``.qhelp`` files must be 
-   converted to ``.md`` before running the analysis. For further information, 
+- ``--sarif-add-query-help``: (supported in version 2.7.1 onwards) adds any custom query help written
+   in markdown to SARIF files (v2.1.0 or later) generated by the analysis. Query help stored in ``.qhelp`` files must be
+   converted to ``.md`` before running the analysis. For further information,
```

Comment:
While we're editing this file. The spacing on lines 70/71 is giving a weird result in the live output: https://codeql.github.com/docs/codeql-cli/analyzing-databases-with-the-codeql-cli/#running-codeql-database-analyze